extensions/amp-base-carousel/1.0/amp-base-carousel.js

```diff
@@ -88,6 +88,14 @@ AmpBaseCarousel['children'] = {
 /** @override */
 AmpBaseCarousel['props'] = {
   'advanceCount': {attr: 'advance-count', type: 'number', media: true},
+  'autoAdvance': {attr: 'auto-advance', type: 'boolean', media: true},
+  'autoAdvanceCount': {attr: 'auto-advance-count', type: 'number', media: true},
+  'autoAdvanceInterval': {
+    attr: 'auto-advance-interval',
+    type: 'number',
+    media: true,
+  },
+  'autoAdvanceLoops': {attr: 'auto-advance-loops', type: 'number', media: true},
```

Comment:
So, it defaults to infinity, but doesn't allow to specify infinity in the attribute. Right? Is that a concern at all? E.g. `auto-advance-loops="(min-width: 400px) 2, Infinity"`

Reply:
Typing in `auto-advance-loops="Infinity"` worked for me. I've updated the AMP storybook knobs to take `text` instead of `number` so we can try things out like this.